articles/connections/database/custom-db/templates/change-password.md

```diff
@@ -304,6 +316,9 @@ function changePassword(email, newPassword, callback) {
 
 ```
 function changePassword(email, newPassword, callback) {
+  var bcrypt = require('bcrypt');
```

Comment:
I only looked at this script only for Mongo and change password, it's quite different from the template we have in https://github.com/auth0/db-connections-templates/blob/139f1598c94fc374ecaef73db3f77ff21aecebb6/src/scripts/mongo/change_password.js#L2

Not quite sure which one is more up-to-date, but should they be synchronized?

Reply:
yes Shu, you are right, all templates in Docs seem to be outdated.
I'll be synchronizing them with the templates in `db-connection-templates`
Thanks!

Reply:
Done
code snippets id docs now are synchronized with the `db-connection-templates`. Some of them were very outdated. 
Thank you Shu!